tray.py

```diff
@@ -127,7 +127,7 @@ def __init__(self, icon_name, scroll_direction):
         self.set_relief(Gtk.ReliefStyle.NONE)
         self.set_size_request(style.GRID_CELL_SIZE, style.GRID_CELL_SIZE)
 
-        icon = Icon(icon_name=icon_name, icon_size=Gtk.IconSize.SMALL_TOOLBAR)
+        icon = Icon(icon_name=icon_name, pixel_size=Gtk.IconSize.SMALL_TOOLBAR)
```

Comment:
Any change that I have to make here?Thanks!

Reply:
Sorry, I missed that.  It is unusual in our code base for an activity to pass a Gtk.IconSize constant instead of a style.*_SIZE constant.  If you have time, please figure out if this is correct.